Detectors/EMCAL/calib/macros/TriggerDCS_CCDBApiTest.C

```diff
@@ -0,0 +1,135 @@
+#include "RStringView.h"
+#include <ctime>
+#include <iostream>
+#include <sstream>
+#endif
+
+// functions getting test data
+void ConfigureTRU(o2::emcal::TriggerTRUDCS& testobject)
+{
+  testobject.setSELPF(7711);
+  testobject.setL0SEL(1);
+  testobject.setL0COSM(100);
+  testobject.setGTHRL0(132);
+  testobject.setMaskReg(1024, 0);
+  testobject.setMaskReg(0, 1);
+  testobject.setMaskReg(512, 2);
+  testobject.setMaskReg(31985, 3);
+  testobject.setMaskReg(0, 4);
+  testobject.setMaskReg(0, 5);
+  testobject.setRLBKSTU(0);
+  testobject.setFw(0x21);
+}
+
+void ConfigureSTU(o2::emcal::TriggerSTUDCS& testobject)
+{
+  testobject.setGammaHigh(0, 0);
+  testobject.setGammaHigh(1, 0);
+  testobject.setGammaHigh(2, 115);
+  testobject.setGammaLow(0, 0);
+  testobject.setGammaLow(1, 0);
+  testobject.setGammaLow(2, 51);
+  testobject.setJetHigh(0, 0);
+  testobject.setJetHigh(1, 0);
+  testobject.setJetHigh(2, 255);
+  testobject.setJetLow(0, 0);
+  testobject.setJetLow(1, 0);
+  testobject.setJetLow(2, 204);
+  testobject.setPatchSize(2);
+  testobject.setFw(0x2A012);
+  testobject.setMedianMode(0);
+  testobject.setRegion(0xffffffff);
+  for (int i = 0; i < 4; i++)
+    testobject.setPHOSScale(i, 0);
+}
+
+/// \brief Converting time into numerical time stamp representation
+unsigned long create_timestamp(int year, int month, int day, int hour, int minutes, int seconds)
+{
+  struct tm timeinfo;
+  timeinfo.tm_year = year;
+  timeinfo.tm_mon = month;
+  timeinfo.tm_mday = day;
+  timeinfo.tm_hour = hour;
+  timeinfo.tm_min = minutes;
+  timeinfo.tm_sec = seconds;
+
+  time_t timeformat = mktime(&timeinfo);
+  return static_cast<unsigned long>(timeformat);
+}
+
+/// \brief Read-write test
+///
+/// Writing to EMCAL CCDB server
+/// Attention: Might overwrite existing CCDB content - use with care!
+void TriggerDCS_CCDBApitest(const std::string_view ccdbserver = "emcccdb-test.cern.ch")
+{
+  std::cout << "Using CCDB server " << ccdbserver << std::endl;
+  o2::ccdb::CcdbApi ccdbhandler;
+  ccdbhandler.init(ccdbserver.data());
+
+  // Prepare database object
+  o2::emcal::TriggerDCS* dcs = new o2::emcal::TriggerDCS;
+
+  o2::emcal::TriggerSTUDCS stuEMCal;
+  ConfigureSTU(stuEMCal);
+
+  o2::emcal::TriggerSTUDCS stuDCal;
+  ConfigureSTU(stuDCal);
+  stuDCal.setRegion(0xffffff7f);
+
+  o2::emcal::TriggerTRUDCS tru;
+  ConfigureTRU(tru);
+
+  dcs->setSTUEMCal(stuEMCal);
+  dcs->setSTUDCal(stuDCal);
+  dcs->setTRU(tru);
+
+  // Set time limits: These are from the start of the run validity range (252235) to the end of the run validity range (267166) LHC16
+  auto rangestart = create_timestamp(2016, 4, 23, 0, 58, 40),
+       rangeend = create_timestamp(2016, 12, 5, 6, 3, 19);
+  std::cout << "Using time stamps " << rangestart << " and " << rangeend << std::endl;
+  std::map<std::string, std::string> metadata;
+  ccdbhandler.storeAsTFile(new o2::TObjectWrapper<o2::emcal::TriggerDCS>(dcs), "EMC/TriggerDCS", metadata, rangestart, rangeend);
```

Comment:
TObjectWrapper is no longer needed! Can you please now use the `ccdbhandler.storeAsTFileAny` interface which allows you to store any type.